contracts/src/v0.8/VRFV2Wrapper.sol

```diff
@@ -343,6 +344,13 @@ contract VRFV2Wrapper is ConfirmedOwner, TypeAndVersionInterface, VRFConsumerBas
     return weiPerUnitLink;
   }
 
+  /**
+   * @dev Calculates premium needed for running an assembly call() post-EIP150.
+   */
+  function getEIP150Overhead(uint32 gas) private pure returns (uint32) {
+    return gas / 64;
```

Comment:
We should be aware that Solidity will round down in cases where `gas` is not divisible by `64`.

Examples:

* `127 / 64 = 1`
* `63999 / 64 = 999`

This could pose issues in rare edge cases where `gas` is fairly low.

Reply:
I think the case where this could actually be a problem would be way lower than any reasonable transaction, but this is a good callout. To be safe and for better form I could replace this with `gas / 64 + 64`?